projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,3379 @@
+	"quark.config.experimental.adjustable_chat.desc": "可以调节聊天栏的水平/垂直偏移。",
+	"quark.config.experimental.adjustable_chat.horizontal_shift.name": "水平偏移",
+	"quark.config.experimental.adjustable_chat.vertical_shift.name": "垂直偏移",
+
+	"quark.config.experimental.climate_control_remover.name": "气候控制移除",
+	"quark.config.experimental.climate_control_remover.disable_continentalness.name": "禁用大陆性",
+	"quark.config.experimental.climate_control_remover.disable_continentalness.desc": "选择要生成的生物群系时，禁用「大陆性」的比较。\n警告：启用此项后，海洋可能会变得更像河流。",
+	"quark.config.experimental.climate_control_remover.disable_depth.name": "禁用深度",
+	"quark.config.experimental.climate_control_remover.disable_depth.desc": "选择要生成的生物群系时，禁用「深度」的比较。\n警告：启用此项后，洞穴生物群系可能会出现在不太常见的高度。",
+	"quark.config.experimental.climate_control_remover.disable_erosion.name": "禁用侵蚀度",
+	"quark.config.experimental.climate_control_remover.disable_erosion.desc": "选择要生成的生物群系时，禁用「侵蚀度」的比较。\n警告：启用此项后，可能会产生非常极端的高度差异，并将使末地更加混乱。",
+	"quark.config.experimental.climate_control_remover.disable_humidity.name": "禁用湿度",
+	"quark.config.experimental.climate_control_remover.disable_humidity.desc": "选择要生成的生物群系时，禁用「湿度」的比较。",
+	"quark.config.experimental.climate_control_remover.disable_offset.name": "禁用偏移",
+	"quark.config.experimental.climate_control_remover.disable_offset.desc": "选择要生成的生物群系时，禁用「偏移」的比较。\n警告：启用此项后，较稀有的下界生物群系会更常见。",
+	"quark.config.experimental.climate_control_remover.disable_temperature.name": "禁用温度",
+	"quark.config.experimental.climate_control_remover.disable_temperature.desc": "选择要生成的生物群系时，禁用「温度」的比较。",
+	"quark.config.experimental.climate_control_remover.disable_weirdness.name": "禁用奇异性",
+	"quark.config.experimental.climate_control_remover.disable_weirdness.desc": "选择要生成的生物群系时，禁用「奇异性」的比较。\n警告：启用此项后……好吧，可能使世界变得奇异。",
+
+	"quark.config.experimental.enchantment_predicates.name": "魔咒谓词",
+	"quark.config.experimental.enchantment_predicates.desc": "针对「要注册的魔咒」中定义的魔咒，此功能为「要更改的物品」中定义的物品生成资源包物品模型谓词。\n\n示例：如果将「minecraft:silk_touch」添加到「要注册的魔咒」中，将「minecraft:netherite_pickaxe」添加到「要更改的物品」中，那么名为「quark_has_enchant_minecraft_silk_touch」的谓词将可用于 netherite_pickaxe.json 物品模型，该谓词的值等于魔咒等级。",
+	"quark.config.experimental.enchantment_predicates.enchantments_to_register.name": "要注册的魔咒",
+	"quark.config.experimental.enchantment_predicates.items_to_change.name": "要更改的物品",
+
+	"quark.config.experimental.enchantments_begone.name": "魔咒禁用",
+	"quark.config.experimental.enchantments_begone.enchantments_to_begone.name": "要禁用的魔咒",
+
+	"quark.config.experimental.game_nerfs.name": "游戏削弱",
+	"quark.config.experimental.game_nerfs.disable_falling_block_dupe.name": "禁用掉落方块复制",
+	"quark.config.experimental.game_nerfs.disable_falling_block_dupe.desc": "使下落的方块无法（利用维度穿梭）复制。",
+	"quark.config.experimental.game_nerfs.disable_ice_roads.name": "禁用冰路",
+	"quark.config.experimental.game_nerfs.disable_ice_roads.desc": "使船无法在冰上滑行。",
+	"quark.config.experimental.game_nerfs.disable_iron_farms.name": "禁用刷铁机",
+	"quark.config.experimental.game_nerf
```

```suggestion
	"quark.config.oddities.matrix_enchanting.allow_short_block_enchanting.name": "允许矮方块",
```
如果需要保留“附魔”两字，那就得改成“含矮方块附魔”这种表述，不大好。

---

其实我感觉 `允许矮方块` 和 `允许矮方块附魔` 意义不明的程度是一样的

---

```suggestion
"quark.config.oddities.matrix_enchanting.allow_short_block_enchanting.name": "矮方块不阻挡附魔",
```
既然都意义不明，那就可以再改得激进点了

---

不错，更准确说
`矮方块不影响附魔`
`矮方块不影响附魔等级`
`矮方块不阻挡有效方块`
`矮方块不阻挡附魔台附近的有效方块`
……

---

`矮方块不影响附魔` 看着比较合理，那前两个建议可以废弃掉了